.gitattributes

```diff
@@ -7,5 +7,5 @@
 *.html text
 *.css text
 *.inc text
-test.desc text
+*.desc text eol=lf
```

Comment:
:pick: Can you add the explanation about the line endings causing problems with `test.pl` to the commit message body please?